src/sema/expression.rs

```diff
@@ -4907,8 +4907,8 @@ fn member_access(
         return Ok(expr);
     }
 
-    // is it a constant in a library (unless basecontract is a local variable)
-    match library_constant(e, id, file_no, ns, symtable) {
+    // is it a constant (unless basecontract is a local variable)
+    match constant(e, id, file_no, ns, symtable) {
         None => (),
         Some(expr) => return Ok(expr),
```

Comment:
This seems like an baroque way of saying:

```
if let Some(expr) =  constant(e, id, file_no, ns, symtable) {
    return Ok(expr);
}
```

What do yo think?

Reply:
Yes, of course!